mlrun/db/httpd.py

```diff
@@ -464,6 +464,7 @@ def get_files():
     user = request.args.get('user', '')
     size = int(request.args.get('size', '0'))
     offset = int(request.args.get('offset', '0'))
+    access_key = request.headers.get('X-V3io-Session-Key')
```

Comment:
Since we have this code twice, maybe create a `get_secrect(request)` function that will return the `secret` dict?